README.md

```diff
@@ -12,6 +12,34 @@ composer require --dev spryker-sdk/composer-constrainer
 
 This is a development only "require-dev" module. Please make sure you include it as such.
 
+Add the console command `\SprykerSdk\Zed\ComposerConstrainer\Communication\Console\ComposerConstraintConsole` to your `\Pyz\Zed\Console\ConsoleDependencyProvider::getConsoleCommands()`.
```

Comment:
Why are not using prefix \ here as that has no additional meaningful value on FQCN here.